src/search/include/search/search.h

```diff
@@ -313,27 +283,31 @@ class TreeSearch
 		if (node == nullptr) {
```

Comment:
How can that happen? If we want to have some default behavior, maybe then we should also set the default value to nullptr for the passed node, or can it also happen in other ways?

Reply:
Yes, the idea is to have a default so we can just call `label()` without any argument. The default is already `nullptr`, I'm not sure what you mean?

Reply:
Nevermind :)